recipes/install-oc-base-packages.rb

```diff
@@ -5,36 +5,19 @@
 
 include_recipe "oc-opsworks-recipes::update-package-repo"
 
-# make sure we get this one from the epel repo or it will cause dep conflicts with tesseract
-install_package("libwebp", %Q|--disablerepo="*" --enablerepo="epel"|)
-
-packages = %Q|mysql57 postfix mailx tesseract|
-
-if mh_node?
-  install_java_11
-  Chef::Log.info "install_java_11" 
-
-  package 'java-1.8.0-openjdk' do
-    action :remove
-    ignore_failure true
-  end
-else
-  packages = packages + " java-1.8.0-openjdk java-1.8.0-openjdk-devel"
-  Chef::Log.info packages
-end
+packages = %Q|rh-mysql57 postfix mailx tesseract libwebp java-11-openjdk java-11-openjdk-devel java-11-openjdk-headless|
 
 install_package(packages)
 
-# remove java-1.7 if needed 
-package 'java-1.7.0-openjdk' do
-  action :remove
-  ignore_failure true
-end
-
-# we use postfix
-package 'sendmail' do
-  action :remove
-  ignore_failure true
+# Something in OC, possibly the spring framework stuff, requires
+# TLSv1.1 to be enabled for the jdbc connection handshake to work,
+# so that's all this file does: removes TLSv1.1 from the list
+# of disabled tls protocols
+cookbook_file 'java.security' do
+  path %Q|/usr/lib/jvm/java-11-openjdk/conf/security/java.security|
+  owner "root"
+  group "root"
+  mode "644"
 end
```

Comment:
There is a StackOverflow on a similar issue where one response was that they upgraded their mysql-connector-java to v5.1.49 to resolve the issue. We are on v5.1.43 and have been since 2017. This is probably because Opencast upstream removed mysql support in later versions and it's up to DCE to update that lib. 
Not sure if this will be the silver bullet for the TSL1.2 issue, but maybe worth trying. https://stackoverflow.com/questions/67332909/why-can-java-not-connect-to-mysql-5-7-after-the-latest-jdk-update-and-how-should/71631180#71631180

There is a Stackoverflow about TLS1.2 
3226732cb0e (Lars Kiesow         2017-08-30 21:59:49 +0200 1250)         <version>5.1.43</version>
```
      <!-- #DCE Use mysql -->
      <dependency>
        <groupId>mysql</groupId>
        <artifactId>mysql-connector-java</artifactId>
        <version>5.1.43</version>
      </dependency>
 ```

Reply:
This rings a bell. I believe we tried that already?

Reply:
We did but I'm going to try one more time. It's possible there was another problem going on when I tried this previously.

Reply:
OK, I tried this again and it actually worked this time. I was able to add `enabledTLSProtocols=TLSv1.2` to the connection string instead of `useSSL=false`. HOWEVER, now I get a warning in the opencast startup output like:

```
WARN: Establishing SSL connection without server's identity verification is not recommended. According to MySQL 5.5.45+, 5.6.26+ and 5.7.6+ requirements SSL connection must be established by default if explicit option isn't set. For compliance with existing applications not using SSL the verifyServerCertificate property is set to 'false'. You need either to explicitly disable SSL by setting useSSL=false, or set useSSL=true and provide truststore for server certificate verification.```

It's just a warning but I don't know if I'm ok just ignoring it. For one thing it's noisy. I'm trying to read up on how to "provide truststore for server certificate verification" and it's all very confusing. I'm accustomed to all this stuff just working.